.github/workflows/graalwasm-photon-micronaut.yml

```diff
@@ -0,0 +1,38 @@
+name: Test GraalWasm Photon Micronaut Demo
+on:
+  push:
+    paths:
+      - 'graalwasm/graalwasm-photon-micronaut/**'
+      - '.github/workflows/graalwasm-photon-micronaut.yml'
+  pull_request:
+    paths:
+      - 'graalwasm/graalwasm-photon-micronaut/**'
+      - '.github/workflows/graalwasm-photon-micronaut.yml'
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  photon-micronaut:
```

Comment:
Minor: We can rename this job to `run` for consistency with the other two workflow files.

Reply:
Well spotted, thanks! Will fix in a follow up PR.

Reply:
See https://github.com/graalvm/graalvm-demos/pull/286/commits/3e141cf374433ba78a7fa1c32d0c55327cd3e051